Source/monster.cpp

```diff
@@ -4568,7 +4568,7 @@ void SpawnGolem(Player &player, Monster &golem, Point position, Missile &missile
 	golem.position.future = position;
 	golem.position.old = position;
 	golem.pathCount = 0;
-	golem.maxHitPoints = 2 * (320 * missile._mispllvl + player._pMaxMana / 3);
+	golem.maxHitPoints = std::max(1, 2 * (320 * missile._mispllvl + player._pMaxMana / 3));

```

Comment:
Golem HP is fixed point like everyone else's, right?
```suggestion
	golem.maxHitPoints = std::max(64, 2 * (320 * missile._mispllvl + player._pMaxMana / 3));
```